<launch>
  <param name="/camera/driver/depth_registration" value="True" />
  
  <node pkg="rbx1_apps" name="object_tracker" type="object_tracker2.py" output="screen">
  
    <remap from="camera_info" to="/camera/depth_registered/camera_info" />
    <remap from="depth_image" to="/camera/depth_registered/image_rect" />

    <rosparam>
       rate: 10
       max_z: 2.0
       goal_z: 0.6
       z_threshold: 0.05
       x_threshold: 0.1
       z_scale: 1.0
       x_scale: 2.0
       max_rotation_speed: 3.0
       min_rotation_speed: 0.5
       max_linear_speed: 0.4
       min_linear_speed: 0.05
    </rosparam>
    
  </node>
</launch>
